### ISSUE TITLE
Black screen

### ISSUE THREAD

**Issue by ph4ultimate** [Anistream/Anistream-Mobile#34]
When ever i want to watch any anime it shows a black screen and doesn't start please fix it 🙏

**Comment by Victor4132** [comment]
I have the same problem to

**Comment by king7kross** [comment]
Why aren't they responding 
It's been 24 days since this problem started they should do something

**Comment by Fixyourapp** [comment]
Same like did they give up on the app

**Comment by king7kross** [comment]
> Same like did they give up on the app

Yeah they took this app from another source and now they are destroying this app

**Comment by XDgamerCZ** [comment]
Why developers don't care about this amazing app? When it was downed by freaking DMCA and also it has been downed from PlayStore, they stopped doing their work. Peoples report bugs and problems and they just don't care, it looks like this app is made now only as enticement for their Discord server where is no discussion abour Anistream. Please developers open your eyes! I don't wanna watch anime on non-phone friendly websites with ton of ads and search anime on every diffirent server or don't have acess to some anime on Crunchyroll and any acess to Funmation in my country.
